imsim/photon_ops.py

```diff
@@ -43,7 +46,12 @@ class LsstOptics(PhotonOp):
         Enable / disable diffraction (default: enabled)
     """
 
-    _req_params = {"telescope": Optic, "band": str, "boresight": CelestialCoord}
+    _req_params = {
+        "telescope": Optic,
+        "band": str,
+        "boresight": CelestialCoord,
+        "camera": None
```

Comment:
This should be `str` I think.  This is supposed to be whatever type is specified in the config file.  None isn't a type.  But in the imsim-config.yaml edit, you made it a string (copying the string in output.camera), so this should be specified as a string here.
Also, FYI, Geri updated how the telescope is specified in his branch (for the same reason -- an Optic cannot really be given in a yaml file).  That's now a str too.